tests/pages/LoginsOverviewPage.test.tsx

```diff
@@ -12,6 +13,11 @@ import { ShowMoreButton } from "@src/components/display/fewlines/ShowMoreButton/
 import { AccountApp } from "@src/pages/_app";
 import LoginsOverviewPage from "@src/pages/account/logins";
 
+Object.defineProperty(window.document, "cookie", {
+  writable: true,
+  value: "flashMessage=Email address has been deleted",
+});
+
```

Comment:
Wouldn't it be simpler (or more readable) to mock `getFlashMessage`?

Reply:
OK fixed 3e97570